src/main/java/uk/gov/hmcts/reform/pip/data/management/models/publication/HeaderGroup.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.pip.data.management.models.publication;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class HeaderGroup {
+
+    private String provenance;
```

Comment:
could we potentaially add the @NotNull annotations here to the required attributes?

Reply:
👍